platform/zephyr/context.cc

```diff
@@ -16,7 +16,7 @@
 
 #include "chre/platform/context.h"
 
-#include <zephyr.h>
```

Comment:
I'd suggest migrating all `<zephyr/zephyr.h>` to `<zephyr/kernel.h>` as well.

Reply:
Sure, may as well while I'm at it.